AdyenCard/Components/Card/CardComponentExtensions.swift

```diff
@@ -58,9 +58,19 @@ extension CardComponent {
 
             submit(data: data)
         } catch {
+            sendEncryptionErrorEvent()
```

Comment:
are we not interested in the actual error message?

Reply:
The error code is the important part.